### DISCUSSION TITLE
Dalsze prace nad UX/UI Wmi Adventure

### DISCUSSION THREAD

**Discussion by mattyl006** [emkarcinos/WMIAdventure#396] [Business plan]
Jak dalej widzimy prace nad designem? Jaka jest dalsza rola Kasi, która nam pomogła? Czy dziękujemy jej już za współpracę i robimy resztę sami, czy coś ma jeszcze dla nas zrobić? Ikony dla kart - prawdopodobnie Michał ze swoją dziewczyną. Grafiki do designu - Grzesław.

**Comment by emkarcinos** [comment]
Z całym szacunkiem dla Kasi, ale jeszcze ani razu nie zrobiła dla nas niczego konkretnego. Zanim w ogóle cokolwiek nam zaproponowała powiedziała, że musimy jej więcej zapłacić. Potem poprosiliśmy ją o design trybu Battle, a wysłała nam landing page. Sam interfejs wyglądał ładnie, ale moim zdaniem nie jest to to, czego szukamy.

**Reply by mattyl006** [reply]
Nie powiedziała że musimy jej więcej zapłacić, tylko zasugerowała, że jeśli miałaby zrobić cały design to wtedy więcej, a my nie chcieliśmy żeby zrobiła całości.

**Reply by mattyl006** [reply]
To lecimy dalej sami tym interfejsem, który ja i @Michael-Czekanski stworzyliśmy w oparciu o ten landing page Kasi? Oczywiście nadal tam potrzeba wielu poprawek, ale czy możemy już iść tym brandingiem? No i grafik brakuje jeszcze of course

**Reply by emkarcinos** [reply]
Dobrze byłoby to przedyskutować z Grzesławem. Rozmawiałem z nim ostatnio i bardzo mu się gryzła walka i zielony, przyjemny kolor

**Reply by itchystone** [reply]
Tak, będę się zajmować kartami.

Faktycznie wtedy bardzo potrzebowaliśmy design battle i niestety nie dostaliśmy go, tylko ten landing page. (w sumie i tak pewnie byśmy nie zdążyli zaimplementować battle)

Też uważam, że interfejs, który mamy teraz jest ładny, ale to nie jest to. Chcieliśmy, aby było to nacechowane informatycznie, brak terminala na pasku bocznym itp. Paleta kolorów też pewnie się trochę zmieni.

Chyba lepiej będzie, jak sami będziemy robić ten design, jako że najlepiej czujemy o jaki klimat nam chodzi. Będziemy też decydować jaką część tego designu tworzymy w danym momencie, jaka nam jest aktualnie potrzebna.

**Reply by mattyl006** [reply]
Ok, brzmi to sensownie co napisałeś @Michael-Czekanski. Mam tylko nadzieję że to co zrobiliśmy dotychczas w Figmie jest w miarę aktualne, przyda nam się i nie pójdzie praca nad tym od początku.

**Reply by itchystone** [reply]
Chyba to co mamy teraz w figmie jest całkiem spoko, dodamy jakoś ten terminal na pasku bocznym i uzupełnimy grafiki i będzie git

**Reply by emkarcinos** [reply]
Moim zdaniem dobrze będzie bardziej zaangażować Grzesława w proces designu UI. Nie chodzi mi tu o to, zeby on cokolwiek nam robił, tylko udostępniać mu to, co robimy i prosić go o wskazówki.
